README.md

```diff
@@ -25,17 +37,25 @@ Hyper Text Markup Language, or HTML, is a way to demarcate a document into diffe
   + Siblings do not inherit from one another, but are important for selecting in CSS.
 
 Here is an example of element relations:
+
 ```html
 <div>  <!-- the parent element -->
   <p></p>  <!-- the first sibling element/the first child-->
   <p></p>  <!-- the second sibling element/the second child -->
 </div>
 ```
 
-## CSS Basics
-Cascading Style Sheets, or CSS, is a language created to style an HTML document by telling the browser how specific elements should look. CSS does this by selecting elements based on their tag, ids, classes, or all of the above. The reason for CSS is the separation of concerns. We want HTML only to be concerned with how it displays and demarcates information, and we let CSS worry about how to make that information look pretty.
+## Review CSS Basics
+
+Cascading Style Sheets, or CSS, is a language created to style an HTML document
+by telling the browser how specific elements should look. CSS does this by
+selecting elements based on their tag, ids, classes, or all of the above. The
+reason for CSS is the separation of concerns. We want HTML only to be concerned
```

Comment:
the adverb makes this sentence a bit confused to my ear.

"We want HTML to focus on the structure and meaning of our content. We let CSS worry about..."